### PR TITLE
Implemented Bring me command

### PR DESCRIPTION
**Pull request by comodoro**
Bring me feature ( "bring me \<something\>", with modifiable \<something\> )

- Works now but with ugly exceptions in the nalink window
- No defaults in settings and does not accept empty list of items, bringme.json from https://gist.github.com/comodoro/922717a4eedd4a5691ae42b41e3d349f has to be copies to caster/bin/data
- Depends on the SelfModifyingRule fix in #271

### CODE REVIEW

**Comment by LexiconCode** [comment]
Very cool :)

**Comment by comodoro** [comment]
Default json file added to `/bin/media` (maybe there is a better location) And removed the `add_selfmodrule` line and unused Grammar import and it seems to work; it still depends on #271 

It is being added in _caster.py as normal rule

By the way the wildcards in .gitignore, particularly *.json for the whole repository, made me figuring out for a while what is going on.

**Comment by LexiconCode** [comment]
For consistency's sake I would leave in  `/bin/data/` instead of `/bin/media`. In the near future we will be moving the user configuration files outside the project folder to a user space.

File: caster/bin/media/bringme.json.defaults [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "Best": "https://www.google.com/",
```
[the remark this answers is not in the document]
Comment (comodoro) [reply caster/bin/media/bringme.json.defaults]: Oh my god, I am so sorry! I will fix this shortly. I swear I will pay more attention from now on.

By the way, this is an indication of how bad my accuracy is. The word I actually said is "test".

**Comment by comodoro** [comment]
Unfortunately opened windows do not get focus most of the time and there does not seem to be a good workaround

**Comment by LexiconCode** [comment]
May be related to https://github.com/synkarius/caster/issues/106?

**Comment by comodoro** [comment]
This is basically solved with the browser detection, I will finish it shortly, maybe after #269

**Pull request merged**